app/src/main/java/com/mapbox/maps/testapp/examples/java/DSLStylingJavaActivity.java

```diff
@@ -0,0 +1,204 @@
+import static com.mapbox.maps.extension.style.expressions.generated.Expression.gt;
+import static com.mapbox.maps.extension.style.expressions.generated.Expression.literal;
+import static com.mapbox.maps.extension.style.expressions.generated.Expression.rgb;
+import static com.mapbox.maps.extension.style.expressions.generated.Expression.subtract;
+import static com.mapbox.maps.extension.style.expressions.generated.Expression.toNumber;
+import static java.text.DateFormat.getDateTimeInstance;
+
+public class DSLStylingJavaActivity extends AppCompatActivity implements OnMapClickListener {
+    public static final ArrayList<ArrayList<Double>> POINT_LIST = new ArrayList<ArrayList<Double>>() {
+        {
+            add(new ArrayList<Double>() {
+                {
+                    add(-35.859375);
+                    add(58.44773280389084);
+                }
+            });
+            add(new ArrayList<Double>() {
+                {
+                    add(-16.171875);
+                    add(58.44773280389084);
+                }
+            });
+            add(new ArrayList<Double>() {
+                {
+                    add(-16.171875);
+                    add(54.7246201949245);
+                }
+            });
+            add(new ArrayList<Double>() {
+                {
+                    add(-35.859375);
+                    add(54.7246201949245);
+                }
+            });
+        }
+    };
+    private static final String IMAGE_URL = "https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Mapbox_logo_2019.svg/2560px-Mapbox_logo_2019.svg.png";
+    private static final String GEOJSON_URL = "https://www.mapbox.com/mapbox-gl-js/assets/earthquakes.geojson";
+    private static final String SOURCE_ID = "earthquakes";
+    private static final String IMAGE_SOURCE_ID = "image";
+    private static final String CIRCLE_LAYER_ID = "earthquakeCircle";
+    private static final String SYMBOL_LAYER_ID = "earthquakeText";
+    private static final String RASTER_LAYER_ID = "raster";
+    private static final Expression MAG_KEY = literal("mag");
+    private static final List<String> QUERY_LIST = new ArrayList() {
+        {
+            add(CIRCLE_LAYER_ID);
+            add(SYMBOL_LAYER_ID);
+        }
+    };
+    public static final ArrayList<String> TEXT_FONT = new ArrayList<String>() {
+        {
+            add("Open Sans Regular");
+            add("Arial Unicode MS Regular");
+        }
+    };
+    private MapboxMap mapboxMap;
+    private MapView mapView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_simple_map);
+        mapView = findViewById(R.id.mapView);
+        mapboxMap = mapView.getMapboxMap();
+        mapboxMap.loadStyle(createStyle(), null, null);
+        MapboxPluginUtils.addOnMapClickListener(mapboxMap, this);
```

Comment:
Debatable but I personally would prefer to use `{NameOfKotlinClass}Utils` for discoverability.
If the original function was found on MapboxMap, I would expect `MapboxMapUtils`
if it was found on MapView, I would expect `MapViewUtils`

wdyt @mapbox/maps-android ?

Reply:
Sounds good to me.

Reply:
Great suggestion @tobrun 👍

Reply:
We should make this a guideline while exposing kotlin extension functions to Java 😄